go.mod

```diff
@@ -135,7 +135,7 @@ require (
 	google.golang.org/grpc v1.23.1
 	gopkg.in/gcfg.v1 v1.2.0
 	gopkg.in/square/go-jose.v2 v2.2.2
-	gopkg.in/yaml.v2 v2.2.4
+	gopkg.in/yaml.v2 v2.2.5
```

Comment:
Is there a reason why we have not pushed the v2.2.6 dependency to the top level but everywhere else?